Panic when calling IssuerVkey() on Byron EBB block

```
panic: runtime error: cannot convert slice with length 0 to array or pointer to array with length 32

goroutine 53 [running]:
github.com/blinklabs-io/gouroboros/ledger/byron.(*ByronEpochBoundaryBlockHeader).IssuerVkey(...)
	/home/agaffney/go/pkg/mod/github.com/blinklabs-io/gouroboros@v0.93.2/ledger/byron/byron.go:494
```

--- a/ledger/byron/byron.go
+++ b/ledger/byron/byron.go
@@ -732,7 +732,7 @@ func (h *ByronEpochBoundaryBlockHeader) SlotNumber() uint64 {
 
 func (h *ByronEpochBoundaryBlockHeader) IssuerVkey() common.IssuerVkey {
 	// Byron blocks don't have an issuer
-	return common.IssuerVkey([]byte{})
+	return common.IssuerVkey{}
 }
 
 func (h *ByronEpochBoundaryBlockHeader) BlockBodySize() uint64 {

